src/app/about/page.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import React, { useState } from "react";
+
+import ConfirmCancelModal from "../../components/cancel-sub-modal/ConfirmCancelModal";
+import { Button } from "../about/button";
+
+const AboutPage = () => {
```

Comment:
this is suppose to be component not a page, please refer to the git issue and read through it carefully, understand the task, tick each checkbox as the you do them and lastly attach a screenshot or a recorded video.
Run tests, if they all pass, take a screenshot and upload it as well.